### ISSUE TITLE
networkit not installable on debian 11 (testing)

### ISSUE THREAD

**Issue by gabri94** [networkit/networkit#535]
Hello, i have tried to install networkit on a fresh virtual environment on my debian testing system, but it failed. Howver i can instal it without any problem on Debian stable (10)
I am running Python 3.8.2.

It fails while:
```Building wheel for networkit```

The output is quite long, so you can find it attached.
[nk_err.log](https://github.com/networkit/networkit/files/4541371/nk_err.log)

**Comment by gabri94** [comment]
I managed to install python3.7 and and select it as runtime for my environment. It compiled.
I can assume that there's a problem with the version 3.8 of python

**Comment by Thijss** [comment]
Can confirm this. Installed Ubuntu 20.04, which ships with python 3.8
Having the same issue. I'll try python 3.7 too.

**Comment by avdgrinten** [comment]
Could this be a Cython issue? Could you install `cython` locally (e.g. via pip3) and try to rebuild it? Does the master branch show the same behavior as the released version of NetworKit?

**Comment by fabratu** [comment]
I can confirm that this is very likely a Cython-error. In Python 3.8 the signature of `PyCode_New()` has [changed](https://github.com/cython/cython/issues/2938). In the PyPI-version of NetworKit we used an older version of Cython, which is not aware of this changes. As suggested by @avdgrinten the problem is solved by rebuilding NetworKit from source using Cython 0.29 and newer:
```
git clone https://github.com/networkit/networkit networkit  
cd networkit  
python3 setup.py build_ext [-jX]  
pip3 install -e .
```

You can do actually build different versions of NetworKit:

`git checkout master` -> version 6.0
`git checkout 6.1` -> version 6.1
(use other tags for older versions)

**Comment by fabratu** [comment]
In the next release, which we plan to release in the upcoming week, this error should be resolved.

**Comment by fabratu** [comment]
Closing this issue - problem is fixed in version 7.0.
